library/src/main/java/com/novoda/downloadmanager/HttpClient.java

```diff
@@ -0,0 +1,23 @@
+package com.novoda.downloadmanager;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+interface HttpClient {
+
+    DownloadManagerResponse execute(DownloadManagerRequest downloadManagerRequest) throws IOException;
+
+    interface DownloadManagerResponse {
+
+        int code();
+
+        boolean isSuccessful();
+
+        String header(String name, String defaultValue);
+
+        InputStream openByteStream() throws IOException;
+
+        void closeByteStream() throws IOException;
+    }
+
```

Comment:
mindful of the project convention re: blank lines - I see a mix in this PR

Reply:
I don't know what the convention is in this project 😂